sys-boot/tboot/tboot-1.10.3.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit mount-boot toolchain-funcs
+
+DESCRIPTION="Performs a measured and verified boot using Intel Trusted Execution Technology"
+HOMEPAGE="https://sourceforge.net/projects/tboot/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+IUSE="custom-cflags selinux"
+
+# requires patching the kernel src
+RESTRICT="test"
+
+DEPEND="|| ( >=dev-libs/openssl-1.1.1l-r1
```

Comment:
This seems equivalent to what it was before? Why change it?

Reply:
CI won't pass without it. As of >=dev-libs/openssl-1.1.1l-r1 (still unstable), the bindist flag is gone. However, there are still versions of openssl with the "bindist" so this construction is needed.

Reply:
What's wrong? Seems to work for me:

```
~/gentoo/gentoo/sys-boot/tboot $ grep openssl tboot-1.10.3.ebuild
DEPEND="dev-libs/openssl[-bindist(-)]"
~/gentoo/gentoo/sys-boot/tboot $ pkgcheck scan
sys-boot/tboot
  UnstableOnly: for arches: [ amd64, x86 ], all versions are unstable: [ 1.10.3 ]
```

Reply:
Ah, that's exactly what I was looking for. I did not know about that construction. I will use that.

Reply:
Sorry, seems I missed the subslot so should be:

```
DEPEND="dev-libs/openssl:0=[-bindist(-)]"
```

Reply:
Done, with subslot operator